### PR TITLE
write out css files to build and dist folders

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -290,13 +290,32 @@ function compile(watch, shouldMinify, opt_preventRemoveAndMakeDir,
...
+    var promise = Promise(r => {
+      resolve = r;
+    });
+    gulp.src('css/**.css')
+        .pipe($$.file('css.js', 'export const cssText = ' +
+            JSON.stringify(css)))
+        .pipe(gulp.dest('build'))
+        .on('end', function() {
+          fs.writeFileSync('build/amp.css');
+          resolve(css);
```
Comment (dvoytenko) [inline gulpfile.js]: Wrap promise around? E.g. 
```
return new Promise(resolve => {
  gulp.src(...)
     .on('end', function() {
       ....
       resolve(css);
    })
})
```

File: gulpfile.js [hunk 2]
```diff
@@ -290,13 +290,32 @@ function compile(watch, shouldMinify, opt_preventRemoveAndMakeDir,
...
+    return promise;
   });
 }
 
 /**
+ * Copies the css from the build folder to the dist folder
+ */
+function copyCss() {
+  fs.copySync('build/amp.css', 'dist/amp.css');
+  return gulp.src('build/**/amp-*.css')
```
Comment (dvoytenko) [inline gulpfile.js]: Are you copying all CSS? Do we really need all of them?

Comment (erwinmombay) [inline gulpfile.js]: currently I am. better all of them though right for consistency? or is it just the main binary that we need?

File: gulpfile.js (hunk 1 above)
Comment (erwinmombay) [inline gulpfile.js]: done.

File: gulpfile.js (hunk 2 above)
Comment (dvoytenko) [inline gulpfile.js]: Yeah. Good point. Let's do.

**Pull request merged**